### PR TITLE
Navigate back when canceling health T&C

### PR DESCRIPTION
**Pull request by bshyong**
Adds a fix to navigate back in the browser history when terms and conditions are canceled instead of going to a hardcoded link. Preserves the use of the hard coded `cancelPath` prop if the history  does not exist.

Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4347

### CODE REVIEW

**Comment by patrickvinograd** [comment]
This doesn't quite work as expected for the T&C modal from the user profile page. If you cancel out of the modal I'd expect it to return you to the profile page, instead it goes back to whatever you were on before the profile page. Since opening the modal from there doesn't push anything onto the history.

**Comment by bshyong** [comment]
Ah, right. That won't work because the prompt shows up in a modal. I will add an override prop which forces the cancel button to be the normal hardcoded link

File: src/js/common/components/AcceptTermsPrompt.jsx [hunk 1]
```diff
@@ -91,7 +101,7 @@ class AcceptTermsPrompt extends React.Component {
           </div>
           <div>
             {submitButton}
-            <a href={this.props.cancelPath} className="usa-button usa-button-outline">
+            <a href="#" onClick={this.onCancel} className="usa-button usa-button-outline">
```
Comment (U-DON) [inline src/js/common/components/AcceptTermsPrompt.jsx]: Should this just be a `button` instead of `a` now?

**Review by U-DON — APPROVED** [review]
LGTM. Minor nitpick.

File: src/js/common/components/AcceptTermsPrompt.jsx (hunk 1 above)
Comment (bshyong) [inline src/js/common/components/AcceptTermsPrompt.jsx]: makes sense—will change this

**Pull request merged**